core/src/main/java/hudson/model/MyViewsProperty.java

```diff
@@ -224,7 +224,7 @@ public ViewsTabBar getViewsTabBar() {
 
     public List<Action> getViewActions() {
         // Jenkins.getInstance().getViewActions() are tempting but they are in a wrong scope
-        return Collections.emptyList();
+        return Collections.singletonList(new NewViewAction());
```

Comment:
Does it work in such way? I see no routing logic to `ModifiableViewGroup#addView`

Reply:
yes, it's work
https://yadi.sk/i/JZQWXkP33Nvd7N
probably becouse of line in ```View#getActions``` 
```  	result.addAll(getOwner().getViewActions()); ```